Python/scoreP3.py

```diff
@@ -60,29 +59,35 @@ def scoreP3(truthDir, testDir, phaseNum='3'):
     testRows = []
     with open(testFile, 'rU') as testFileObj:
         try:
-            testReader = csv.DictReader(testFileObj,
-                                        fieldnames=['image', 'confidence'])
+            testReader = csv.DictReader(
+                testFileObj,
+                fieldnames=['image_id', 'melanoma', 'seborrheic_keratosis'])
```

Comment:
Since we specify the field names, `DictReader` will always read the first row, even if it's a header (which we should consider to be optional for submissions). Logic should be added here to skip the first row if it looks like a header.